kpatch-build/create-diff-object.c

```diff
@@ -161,17 +155,16 @@ static int is_bundleable(struct symbol *sym)
  * the object file.  The local entry point is 8 bytes after the global entry
  * point.
  */
-static int is_gcc6_localentry_bundled_sym(struct symbol *sym)
-{
-	return ((PPC64_LOCAL_ENTRY_OFFSET(sym->sym.st_other) != 0) &&
-		sym->sym.st_value == 8);
-}
-#else
-static int is_gcc6_localentry_bundled_sym(struct symbol *sym)
+static bool is_gcc6_localentry_bundled_sym(struct kpatch_elf *kelf,
+					  struct symbol *sym)
 {
+	if (kelf->target_arch == PPC64) {
```

Comment:
I think the 'target_' prefix in 'target_arch' is redundant since the 'kelf' object already represents the target